### ISSUE TITLE
Sudden Frame Spikes when resetting Arena and stuff

### ISSUE THREAD

**Issue by pixeledge-verify** [diamond-rip/Eden#410]
The players including me are getting sudden Frame spikes resetting the arena in round based modes and also some times for example I am playing Boxing with my friend but there is a BuildUHC Match going on at the same time. What happens is the Boxing match ends and the Arena gets reset and BuildUHC player's blocks are gone too. (ImanitySpigot 3 Latest)

**Comment by HierarchyMan** [comment]
your arenas must be very close to the spawn
fram spikes are compeletely client sided so if the arenas arent restting while being loaded by the clienbt you shouldnt get frame spikes

**Comment by pixeledge-verify** [comment]
> your arenas must be very close to the spawn fram spikes are compeletely client sided so if the arenas arent restting while being loaded by the clienbt you shouldnt get frame spikes

my arenas are in different world

**Comment by HierarchyMan** [comment]
Yeah there can only be network lag for the client when the server is stressing, client can't lag is server is having tps drops. 
There must be a different issue most likely not related to Eden

**Comment by GoodestEnglish** [comment]
> The players including me are getting sudden Frame spikes resetting the arena in round based modes and also some times for example I am playing Boxing with my friend but there is a BuildUHC Match going on at the same time. What happens is the Boxing match ends and the Arena gets reset and BuildUHC player's blocks are gone too. (ImanitySpigot 3 Latest)

BuildUHC arena should be separate with boxing arena. They cannot be the same.

About the frame spikes, I will check about it

**Comment by GoodestEnglish** [comment]
Can't reproduce. I believe it is most likely something caused in your side because this is the first time I've ever heard of this.

If you have `set-block-fast` disabled in config, try turn it on. This skips some bukkit api action (Not sure if this will fix it or not)

**Comment by ssquadteam** [comment]
btw uh the %eden_match_unranked/ranked_<kit name>% placeholder is bugged
It show's Player isn't in a Match. Would be great if you could fix it quick:D

**Comment by ssquadteam** [comment]
also for some reason when using CarbonSpigot the TNT's doesn't break the Endstone (Doesn't happen in our BedWars server) and also Arenas dont reset correctly.

**Comment by GoodestEnglish** [comment]
> also for some reason when using CarbonSpigot the TNT's doesn't break the Endstone (Doesn't happen in our BedWars server) and also Arenas dont reset correctly.

Using carbon 3.5 and it works for me
![image](https://github.com/diamond-rip/Eden/assets/45838505/ddf47450-5167-483b-9117-4a430233baec)


> btw uh the %eden_match_unranked/ranked_% placeholder is bugged It show's Player isn't in a Match. Would be great if you could fix it quick:D

Will fix soon. Next time please open another ticket for bugs which isn't related to this issues.
